syntax_and_semantics/visibility.md

```diff
@@ -118,16 +118,45 @@ end
 Namespace::Bar.new.bar
 ```
 
-A `protected` class method can be invoked from an instance method and the other way around:
+A `protected` method can only be invoked from the scope of its class or its descendants. That includes the class scope and bodies of class methods and instance methods of the same type the protected method is defined on, as well as all types including or inherinting that type and all types in that namespace.
 
 ```crystal
-class Person
-  protected def self.say(message)
-    puts message
+class Parent
+  protected def self.protected_method
   end
 
-  def say_hello
-    Person.say "hello" # OK
+  Parent.protected_method # OK
+  
+  def instance_method
+    Parent.protected_method # OK
+  end
+  
+  def self.class_method
+    Parent.protected_method # OK
+  end
+end
+
+class Child < Parent
+  Parent.protected_method # OK
+  
+  def instance_method
+    Parent.protected_method # OK
+  end
+  
+  def self.class_method
+    Parent.protected_method # OK
+  end
+end
+
+class Parent::Sub
```

Comment:
I mean replacing one of this classes by a module. There are already 3 classes

Reply:
You know it, but new comers won't know systematically.

Reply:
They won't learn it either when this line is changed to `module`. On the contrary, it might create confusion.